contracts/SpokePool.sol

```diff
@@ -38,4 +198,12 @@ contract SpokePool {
         uint256[] memory relayerRefundAmounts,
         bytes32[] memory inclusionProof
     ) public {}
+
+    /**************************************
+     *           VIEW FUNCTIONS           *
+     **************************************/
+
+    function chainId() public view returns (uint256) {
+        return block.chainid;
```

Comment:
+1. Hopefully this is cheaper than reading a variable from storage (seems likely!),